dspace-server-webapp/src/test/java/org/dspace/app/rest/CommunityAdminGroupRestControllerIT.java

```diff
@@ -91,7 +82,8 @@ public void getCommunityAdminGroupTest() throws Exception {
     @Test
     public void getCommunityAdminGroupTestCommunityAdmin() throws Exception {
         context.turnOffAuthorisationSystem();
-        Group adminGroup = communityService.createAdministrators(context, parentCommunity);
+        Group adminGroup = GroupBuilder.createCommunityAdminGroup(context, parentCommunity).build();
+        // TODO: this should actually be "add member", not directly setting a policy, right?
```

Comment:
Just for the record, I agree with this TODO.  This may need more cleanup, but we could do this in a separate PR.  It seems like this test is flawed and should actually add the EPerson as a member of the Community Admin group and then make sure that EPerson can see the Community Admin group.

However, now that I type that out, maybe both tests would be useful here.  Verifying someone in the group can see it, and also verifying someone with explicit ADMIN permissions (i.e. this policy creation) can also see that group.